yolo-object-detection.js

```diff
@@ -83,16 +83,19 @@ module.exports = (RED) => {
                 port: '8888',
                 method: 'POST',
                 headers: {
-                    'Content-Length': Buffer.byteLength(msg.payload)
                 }
             }
 
+            let requestBody = null
             try {
-                JSON.parse()
-                options.headers['Content-Type'] = 'application/json'
-                node.debug('Forwarding JSON payload')
-            } catch (e) {
                 node.debug('Forwarding raw image payload')
```

Comment:
Hang on - what do you expect to throw an exception in this `try` block?

Since the existing examples may send a raw image buffer OR a JSON object, the original logic was:

1. Try to parse the payload - if parse throws an exception, then it's a raw image buffer - so forward that.
2. If parse _doesn't_ throw an exception, then it's a JSON payload, so forward that with the right content type.

It seems like you've flipped the logic here - I think you need to put the catch block logic you have back in the `try` block and vice versa.

Reply:
@bleggett - The original logic didn't quite work.  Noticed that the buffer byte length header content-length was throwing an exception when receiving JSON payload.  As a result shifted that logic down to detect whether the msg.payload is a byte buffer and then if not to throw and default to JSON.  Tested it out in the flow and confirmed that it works.  I'm open to different logic to detect.  If this logic is okay, noticed need to shift the node.debug to output at the end of each block.

Reply:
@bleggett - Updated logic for explicit video frame check as discussed.  Please review/approve.  Thanks.